tests/integration/ports/docker-compose.yml

```diff
@@ -6,9 +6,8 @@ services:
       command: ["dumb-init", "/bin/busybox", "httpd", "-f", "-h", "/var/www/html", "-p", "8001"]
       working_dir: /var/www/html
       ports:
-        - 8001:8001
-      volumes:
-        - ./test1.txt:/var/www/html/index.txt:ro,z
```

Comment:
I think that if we're removing custom index.txt, then we may as well replace httpd with something more obvious, like `sleep infinity`. Then `working_dir` can be removed as well. `down_cmd` no longer needs `--volumes`, but needs `-t0` to make test not take 10 seconds more.

Reply:
I've simplified the compose file.

The `-t0` argument is not needed here, because `dump-init` is used as PID1 and so SIGTERM works when shutting down the container.